### PR TITLE
Updating changelog about engine icon

### PR DESCRIPTION
**Pull request by thopiekar**
Added more details about the work I did. Additionally, I think your effort in designing the final icon should be also mentioned!
I'm left-handed with UI designing as you know..

### CODE REVIEW

File: plugins/ChangeLogPlugin/ChangeLog.txt [hunk 1]
```diff
@@ -66,7 +66,7 @@ Generate a cube mesh to prevent support material generation in specific areas of
 *Real bridging - smartavionics
 New experimental feature that detects bridges, adjusting the print speed, slow and fan speed to enhance print quality on bridging parts.
 
-*Updated CuraEngine executable - thopiekar & Ultimaker B.V.
+*Updated CuraEngine executable - thopiekar & Ultimaker B.V. ❤️
```
Comment (Ghostkeeper) [inline plugins/ChangeLogPlugin/ChangeLog.txt]: :heart:

**Comment by thopiekar** [comment]
But please note: I haven't tested, whether the emoji is correctly displayed.
But hey, maybe another situation I managed to get a bug into Cura 😂

**Comment by Ghostkeeper** [comment]
I did test that specifically :stuck_out_tongue:

**Comment by thopiekar** [comment]
Haha, nice :+1:

**Pull request merged**